ch.html

```diff
@@ -9,10 +9,10 @@
     <title>Alphatron</title>
     <link href="/images/favicon.ico" rel="icon" type="images/x-icon">
     <meta charset="utf-8"/>
-    <meta content="Alphatron computer science competition for students aged between 8 and 16" name="title"/>
-    <meta content="Alphatron is a computer science competition for those students in primary or secondary education, that aren't afraid of challenges and like fun puzzles and problems. Taking part is free and the contestants can win valuable prizes."
+    <meta content="Alphatron Informatikwettbewerb für SchülerInnen zwischen 8 und 16 Jahren" name="title"/>
+    <meta content="Alphatron ist ein Informatikwettbewerb für SchülerInnen der Primar- und Sekundarstufe, die keine Angst vor Herausforderungen haben und Spass an Rätseln und Problemen haben. Die Teilnahme ist kostenlos und die TeilnehmerInnen können tolle Preise gewinnen."
           name="description"/>
-    <meta content="programming, computer science, computer, science, competition, contest, prize, win, problem, puzzle, thinking, challenge, challenging, smart, student, pupil, child, children, school, fun, primary school, secondary school, studies, free, interesting, fascinating"
+    <meta content="Programmieren, Informatik, Computer, Wissenschaft, Wettbewerb, Preis, Gewinn, Problem, Rätsel, Denken, Herausforderung, herausfordernd, schlau, Student, Schüler, Kind, Kinder, Schule, Spass, Primarschule, Sekundarschule, Studium, Lernen, kostenlos, interessant, faszinierend"
           name="keywords"/>
     <meta content="Alphatron" name="author"/>
     <meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport"/>
```

Comment:
You might want to change the Sending message and Send attrivutes to getman on lines 31 and 38

Reply:
Similarly on lines 39 and 43

Reply:
[Off-topic]
My apologies, that form is extremely hacky, I am not a front-end developer and I made the initial Hungarian site without any regard to scalability.
We might want to replace that form in the future

Reply:
Done.